.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  publish:
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout
+      - run:
+          name: Install Prereqs
+          command: sudo apt-get update && sudo apt-get install -y yarn
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/.npmrc
+      - run:
+          name: Publish package
+          command: |
+            if ./.circleci/check_npm_version.sh; then
+                yarn publish --access public
```

Comment:
there was one thing i liked about `np`, which was tagging and pushing tags to the repo. is this something we should care about and support?

Reply:
We do this in protocol, however, it is a bit tricky to set up since ci needs to have access to _push_ to the repo.

https://github.com/UMAprotocol/protocol/blob/c6729e94d65be2ccd9bf748bce278e20a4ed73ad/.circleci/config.yml#L84-L86

I can add this.

Reply:
ok yea i was worrired it might be tricky, i dont think its totally necessary, was just a nice thing to have